### PR TITLE
Rachel look at the new code I wrote :)

### PR DESCRIPTION
**Pull request by lrperlmu**

### CODE REVIEW

File: appside/index.html [hunk 1]
```diff
@@ -2,12 +2,22 @@
 <html>
 
 <body>
+  <div id="crumbs"></div>
```
[the remark this answers is not in the document]
Comment (lrperlmu) [reply appside/index.html]: fixed in newest commit!

File: appside/index.js [hunk 2]
```diff
@@ -1,7 +1,103 @@
 "use strict";
 
-let data = knowledgebase_data;
+
+let data = KNOWLEDGEBASE_DATA;
 console.log(data);
 
+let emotions = ['Anger', 'Disgust', 'Envy', 'Fear', 'Guilt', 'Jealousy', 'Sadness', 'Shame']
```
[the remark this answers is not in the document]
Comment (lrperlmu) [reply appside/index.js]: added in newest commit!

File: appside/index.js [hunk 3]
```diff
@@ -1,7 +1,103 @@
...
+let selected_emotion = 'Anger';
+
+$(document).ready( function() {
+    // bread crumbs link to other emotions
+    let crumbs = $('#crumbs');
```
[the remark this answers is not in the document]
Comment (lrperlmu) [reply appside/index.js]: done!

**Comment by sittr7** [comment]
Ship it! :shipit:

**Pull request merged**